### PR TITLE
OPC Publisher Azure integration sample

### PR DESCRIPTION
**Pull request by mregen**
I followed advise in #51 and converted the project to use the maven build to pull in the Azure IoTHub Java support. Also the project is moved to a 'azure-integration' folder. This may not be the final location but it would only require a small move of the root folder once #54 is implemented.

Thanks,
Martin

### CODE REVIEW

File: azure-integration/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (mregen) [inline azure-integration/.gitignore]: Can we call the root directory 'Publisher'?

Comment (bjakke) [inline azure-integration/.gitignore]: I'm not exactly sure what you mean by this?

**Comment by bjakke** [comment]
I will pull this after the existing samples are moved to a sub-folder, probably during the next week.

File: azure-integration/.gitignore (hunk 1 above)
Comment (mregen) [inline azure-integration/.gitignore]: I mean call it /Publisher/... instead of /azure-integration/... ?

Comment (bjakke) [inline azure-integration/.gitignore]: I would prefer a specific name if this example was for publishing data to azure? Also in my opinion lowercase folder names should be used as the other folders are also in lowercase. 

Hmm.. I guess e.g. 'azure-publisher' would be better than 'azure-integration'. I'm not that good at naming things :).

Comment (mregen) [inline azure-integration/.gitignore]: @bjakke azure-publisher sounds good!

**Comment by dk2129** [comment]
Shouldn't it be called 'amqp-publisher'? Or does it include code (other than connection information) that only works with azure?

**Comment by mregen** [comment]
The sample is tied to Azure IotHub. It uses the Azure iothub sdk for Java to publish to the Azure IoTHub service,

**Comment by bjakke** [comment]
I will merge this now and then do the renaming to 'azure-publisher'

**Pull request merged**